### PR TITLE
Fixes for wsrep suite

### PR DESCRIPTION
**Pull request by mkaruza**
* Disabled wsrep-on for all tests in wsrep suite.
* For test which need wsrep-on=ON additional .cnf file is created and .opt file removed.

### CODE REVIEW

File: mysql-test/suite/wsrep/t/mdev_7798.test [hunk 1]
```diff
@@ -1,3 +1,4 @@
+--source include/have_innodb.inc
```
Comment (SachinSetiya) [inline mysql-test/suite/wsrep/t/mdev_7798.test]: Not needed

File: mysql-test/suite/wsrep/t/mdev_7798.test [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
 --source include/have_wsrep_provider.inc
 --source include/have_binlog_format_row.inc
```
Comment (SachinSetiya) [inline mysql-test/suite/wsrep/t/mdev_7798.test]: This also not required

File: mysql-test/suite/wsrep/t/mdev_6832.test [hunk 3]
```diff
@@ -1,3 +1,4 @@
+--source include/have_innodb.inc
```
Comment (SachinSetiya) [inline mysql-test/suite/wsrep/t/mdev_6832.test]: Not needed

**Comment by mkaruza** [comment]
Sachin, have_innodb.inc is needed. 

Reason is that wsrep, if enabled, internally creates wsrep_schema.cluster table on bootstrap.

**Pull request merged**